pkg/inspect/inspect.go

```diff
@@ -44,23 +44,23 @@ type HostConfig struct {
 	ShmSize              int64                       `json:"ShmSize"`
 	Runtime              string                      `json:"Runtime"`
 	ConsoleSize          *specs.Box                  `json:"ConsoleSize"`
-	Isolation            string                      `json:"Isolation"` //TODO
-	CPUShares            *uint64                     `json:"CPUSShares"`
+	Isolation            string                      `json:"Isolation"`  //TODO
```

Comment:
I think this is windows-only. @rhatdan Should we throw this out?

Reply:
Yes drop Isolation.